tests/modules/users/resources/test_getting_users_info.py

```diff
@@ -111,3 +112,49 @@ def test_getting_user_id_not_found(flask_app_client, regular_user):
     ):
         response = flask_app_client.get('/api/v1/users/wrong-uuid')
         assert response.status_code == 404
+
+
+def test_getting_sightings_for_user(flask_app_client, db, staff_user):
+
+    from tests.modules.sightings.resources import utils as sighting_utils
+
+    temp_owner = utils.generate_user_instance(
+        email='user_4_sightings@localhost',
+        is_researcher=True,
+    )
+
+    data_in = {
+        'encounters': [{}],
+        'context': 'test',
+        'locationId': 'test',
+    }
+
+    sighting_create_response = sighting_utils.create_sighting(
```

Comment:
This should really have been a comment on @naknomum s PR that put this in but I would have thought that data_in would be much more likely to be overridden than expected_status_code so it should have come before the status code

Reply:
yeah that makes sense to me especially since i dont fully understand the implication this in python. :)   i can make this change (and the same to _update!) in my pr 121

Reply:
I'm afraid I don't see the consequences here either. Since they are named arguments instead of positional, I expect the same for the input.

Reply:
@colinwkingen If the order is swapped this can go from:
sighting_create_response = sighting_utils.create_sighting(
         flask_app_client, temp_owner, expected_status_code=200, data_in=data_in
    )
to
sighting_create_response = sighting_utils.create_sighting(flask_app_client, temp_owner, data_in)
Arguments that hardly ever use the default should be before arguments that mostly do, It can make the calling code much clearer

Reply:
is good to know this subtlety ... esp going forward when creating new args lists.  thanks for checking up on this.